crates/sui-indexer-alt-jsonrpc/src/data.rs

```diff
@@ -28,11 +29,25 @@ pub(crate) async fn load_live(
         return Ok(None);
     }
 
-    let object = ctx
-        .kv_loader()
-        .load_one_object(object_id, latest_version.object_version as u64)
-        .await
-        .context("Failed to load latest object")?;
+    // Read from kv store and retry if the object is not found.
+    let mut object = None;
+    let retry_interval = Duration::from_millis(ctx.config().objects.kv_retry_interval_ms);
```

Comment:
Same suggestion as above to use `interval` here, instead of just `sleep` -- it's more relevant here as well because the call to `load_one_object` will definitely introduce a delay as the data loader pauses to gather its batch.